dist/profile/templates/kubernetes/resources/chatbot_jenkinsadmin/secret.yaml.erb

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: jenkinsadmin
+  namespace: chatbot
+data:
+  .github: |
```

Comment:
There must be some YAML feature being used here that I'm not aware of, why does the key have a leading `.`?

Reply:
It's not a YAML feature but a kubernetes specificity, a secret can either be mounted as a environment variable or mounted as a volume into a pod. If it's mounted as volume, by default it mounts all secret information into a mounted directory where each secret key represent a filename and the file contents is defined by the value. In this case .github is mounted in /home/chatbot/.github. But we can still decided to only mount several keys or change the destination filename, which is what I will do as I am moving the nick_password to that secret and I don't need to mount it as file